content/blog/debugging-dde.md

```diff
@@ -0,0 +1,149 @@
+---
+author: "Wes Hinsley"
+date: 2019-04-29
+title: Debugging at the edge of reason
+best: false
+---
+
+What do you do if this code...
+```
+printf("Before loop\n");
+while ((a>0) && (b>0)) {
+  printf("Inside loop\n");
+  // Other loop code
+}
+if (a>0) printf("a>0\n");
+if (b>0) printf("b>0\n");
+if ((a>0) && (b>0)) printf("Both a>0 and b>0!");
+
+```
+...produces this output?
+```
+Before loop
+a>0
+b>0
+Both a>0 and b>0!
+```
+
+This, simplified, was a situation we were in when 
```

Comment:
You might want to point out the problem is that that the `inside loop` statement never printed

Reply:
Yep, added a sentence for that.